components/unknown-type-panel.js

```diff
@@ -56,15 +56,23 @@ export default function UnknownTypePanel({ item, attribution = null }) {
         <DataArea>
           <CommonDataRenderer item={item}>
             {_.sortBy(genericProperties).map((property) => {
-              const label = profile?.properties[property]?.title || property;
-              return (
-                <Fragment key={property}>
-                  <DataItemLabel>{label}</DataItemLabel>
-                  <DataItemValue>
-                    <UnspecifiedProperty property={item[property]} />
-                  </DataItemValue>
-                </Fragment>
-              );
+              if (!Array.isArray(item[property])) {
```

Comment:
Does this mean we don't display the property if it is an array?

Reply:
Yes. The generic pages are just to get an idea of what's in the object.